src/pyaro_readers/eeareader/EEATimeseriesReader.py

```diff
@@ -577,9 +628,11 @@ def _country_code(self, country: str) -> str | None:
     @functools.cached_property
     def _country_code_mappings_eea(self) -> dict[str, str]:
         mappings = self._country_code_mappings.copy()
-        mappings.update({
-            "United Kingdom": "GB",
-        })
+        mappings.update(
+            {
+                "United Kingdom": "GB",
```

Comment:
Is this needed in addition to `_country_code_mappings`?

Reply:
I think the `StationFilter` is supposed to filter on ISO codes, but it might be easier to special case UK/GB